.devcontainer/setup.sh

```diff
@@ -20,5 +24,12 @@ fi
 (cd server && POETRY_VIRTUALENVS_CREATE=false poetry install)
 (cd client && npm install)
 
+# Run the database in the background
+docker compose up -d
+
+# Wait for the database to start
+# 5 seconds should be enough
+sleep 5
```

Comment:
This wasn't enough when building from scratch on windows. Should look into doing a polling check like we do in `docker/server/entrypoint.sh`